Core/Types/CkanModule.cs

```diff
@@ -292,6 +295,13 @@ public CkanModule(string json, IGameComparator comparator)
         {
             _comparator = comparator;
 
+            var errors = ValidateAgainstSchema(json);
+            if (errors.Result.Any())
+            {
+                var message = "Validation against the schema failed.\n" + string.Join("\n  ", errors);
+                throw new BadMetadataKraken(null, message);
```

Comment:
I'd like to test the impact of this exception on the GUI. What would be a good way to do that? I tried corrupting my `registry.json` file in various ways, but I think those tests got caught by other checks (e.g., invalid licenses are already covered elsewhere).